test/screens/v2/departure_test.exs

```diff
@@ -472,4 +472,60 @@ defmodule Screens.V2.DepartureTest do
              } == Departure.encode_params(params)
     end
   end
+
+  describe "build_params_for_schedules/2" do
+    test "returns params string route_type when no opts is provided" do
+      params = %{route_type: :subway}
+      opts = []
+
+      result = Departure.build_params_for_schedules(params, opts)
+
+      assert result == %{route_type: [:subway]}
+    end
+
+    test "returns params list route_type  when no opts is provided" do
+      params = %{route_type: [:light_rail, :subway]}
```

Comment:
Technically for a user of the module to create this condition, they'd have to violate the `params` typespec; `route_type` can only be a single value. (Would it work anyway if you passed multiple? If so, maybe we could just update the typespec.)